deep-security-agent/meta/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+dependencies: []
+galaxy_info:
+  description: Install, Activate and Manage Trend Micro Deep Security Agent
+  min_ansible_version: 2.4
+  company: Trend Micro
+  author: Deep Security
+  license: MIT
+  galaxy_tags:
+    - trend-micro
+    - deep-security
+    - dsm
+    - dsa
+  platforms:
+    - name: Amazon
+      versions:
+      - 2016.09
+    - name: Ubuntu
+      versions:
+      - 12.04.5
+      - 14.04.5
+      - 16.04.0
+    - name: Debian
+      versions:
+      - 7
+      - 8
+    - name: RHEL
+      versions:
+      - 6.9
```

Comment:
For redhat or other linux distribution, I suppose we don't need to consider the minor version, you may use the same wording in Windows, ex. "ALL", or just list the major version is enough.

Reply:
Just use the major version. If we just use the wording "ALL" that would be dangerous, because, for example, it's difficult to run our playbook in RHEL5/CentOS5/OracleLinux5.